install.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -ex
+
+sudo apt-get -y update
+sudo apt-get -y install build-essential python3-dev
+sudo apt-get -y install python-pip
+
+sudo apt-get -y install postgresql postgresql-contrib libpq-dev
+sudo service postgresql start
+
+sudo -u postgres createuser -s -d assessor
+sudo -u postgres psql -c "ALTER USER assessor WITH PASSWORD 'assessor';"
+sudo -u postgres createdb assessor
+sudo -u postgres psql -c "grant all privileges on database assessor to assessor"
+
+sudo pip install virtualenv
+
+git clone https://github.com/codefornola/assessor-scraper.git
+
+virtualenv -p python3 venv_scraper
+
+. venv_scraper/bin/activate
+
+pip install requests
```

Comment:
Why not install with `pip install -r requirements.txt` here?

Reply:
Oh, this was because I couldn't get the requirements to actually pip install and this was the subset of libraries that I could install and then get everything to still work!